packages/manager/apps/hpc-vmware-managed-vcd/mocks/vcd-organization/vcd-datacentre.mock.ts

```diff
@@ -18,8 +17,7 @@ export const datacentreList: IVcdDatacentre[] = [
     },
     currentTasks: [],
     targetSpec: {
-      description:
-        'Pour vous proposer une description adaptée, pourriez-vous me donner plus de détails sur votre service ? Cela me permettra de créer une description précise et impactante qui tient dans la limite des 255 caractères.',
+      description: 'VDC Description',
```

Comment:
Value should be different from `currentState.description` to avoid missing regression by using `currentState.description` instead of `targetSpec.description`